source/wp-content/themes/wporg-developer-2023/inc/block-hooks.php

```diff
@@ -24,6 +24,13 @@ function filter_search_block( $block_content, $block ) {
 		if ( isset( $block['attrs']['className'] ) && strpos( $block['attrs']['className'], 'wporg-filtered-search-form' ) ) {
 			$block_content = str_replace( '</form>', do_blocks( '<!-- wp:wporg/search-filters /-->' ) . '</form>', $block_content );
 		}
+		if ( is_singular( 'command' ) ) {
```

Comment:
What do you think about adding this as an else-if clause to flatten the code? 

Maybe something like:

```
if ( is_singular( 'command' ) )  {
	// ...
} else if ( wporg_handbook() ) {  // flip this logic
	// ..
} else {
	// ...
}
```

Reply:
I also wonder if `is_singular` is too specific. What if a search bar was added to an `archive` page? Maybe just do it based on the post type?

Reply:
yeah, you make a good point. I didn't consider that scenario, thanks for bringing it up.